src/connectedk8s/azext_connectedk8s/_help.py

```diff
@@ -123,9 +123,9 @@
   short-summary: Collects diagnose infomation and  gets logs on the connected cluster.
   examples:
       - name: using storage account name and a shared access signature token with write permission
-        text: az connectedk8s troubleshoot -g MyResourceGroup -n ConnectedCluster --storage-account MyStorageAccount --sas-token "MySasToken"
+        text: az connectedk8s troubleshoot -g MyResourceGroup -n ConnectedCluster --storage-account MyStorageAccount --sas-token "MySasToken" --container-name "mycontainername"
```

Comment:
Probably, keep storage-container-name

otherwise, it creates a confusion with container on k8s cluster.